molo/core/tests/test_views.py

```diff
@@ -1794,12 +1801,12 @@ def test_next_article_not_main_language(self):
         response = self.client.get(
             '/sections-main-1/section-a/article-c-in-french/')
         self.assertEqual(response.status_code, 200)
-        self.assertContains(response, self.article_b.title + ' in french')
+        self.assertContains(response, self.article_c.title + ' in french')
```

Comment:
These changes are incorrect. The test is supposed to check that the Next Article feature is working properly. So it should be testing that the article_b title appears somewhere on the page.